runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -1223,6 +1223,15 @@ impl<'a> VMLogic<'a> {
         self.gas_opcodes(gas)
     }
 
+    // TODO!!! Hide this behind a feature flag!
+    pub fn sleep_nanos(&mut self, nanos: u64) -> Result<()> {
+        let duration = std::time::Duration::from_nanos(nanos);
+        eprintln!("boom sleeping for {duration:?}!");
```

Comment:
The host functions are already traced, so these should probably go, or at least be replaced with `tracing::warn` or somesuch.

Reply:
Ah I didn't mean to merge that, it's just my debug logs. I'll remove that.